resources/cancer2EFO_mappings.tsv

```diff
@@ -0,0 +1,75 @@
+Cancer_type_acronym	Cancer_type_name	EFO_id	EFO_name	Source

```

Comment:
Is the `Source` column needed? If so, does it need to be an URL?

Reply:
I think it is good to include the source for an easier understanding in the future. But yes, I can definitely change the URLs to a description.